src/main/features/response/views/macro/expandableSection.njk

```diff
@@ -0,0 +1,13 @@
+{% macro expandableSection(label = '', content = '', limit = 400) %}
+  <div class="show-more">
+    <div class="bold-small">{{ t(label) }}</div>
+    {% if content.length > limit %}
+      <input id="more-less" type="checkbox">
```

Comment:
The ID should be unique across the page otherwise we get unexpected behaviour when more the one 'expandable section' is added to the page.

Reply:
Also can we add something to the name that suggest it holds the state of that expandable block?